auth/auth_test.go

```diff
@@ -86,25 +85,61 @@ func TestMain(m *testing.M) {
 	if err != nil {
 		log.Fatalln(err)
 	}
-	client.ks = ks
+	client.keySource = ks
 
 	testGetUserResponse, err = ioutil.ReadFile("../testdata/get_user.json")
```

Comment:
Would it make sense to break this one out, too?

Reply:
This gets used around 3 times in the test suite. We can avoid a few additional file system reads by doing it here.